go.mod

```diff
@@ -2,14 +2,13 @@ module github.com/Qovery/pleco
 
 go 1.16
 
-replace github.com/Qovery/pleco/cmd => ./cmd
-
-replace github.com/Qovery/pleco/core => ./core
-
-replace github.com/Qovery/pleco/providers/aws => ./providers/aws
-
-replace github.com/Qovery/pleco/providers/k8s => ./providers/k8s
-
-replace github.com/Qovery/pleco/utils => ./utils
+replace (
```

Comment:
Just to be sure, it is something which should be commited? Or is it just for dev?